srcpkgs/goattracker-stereo/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'goattracker-stereo'
+pkgname=goattracker-stereo
+version=2.76
+revision=1
+wrksrc=gt2stereo
+makedepends="SDL-devel"
+short_desc="C64 music tracker (stereo)"
+maintainer="Zeinok <zeinok@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://sourceforge.net/projects/goattracker2/"
+distfiles="${SOURCEFORGE_SITE}/goattracker2/GoatTracker%202%20Stereo/${version}/GoatTracker_${version}_Stereo.zip"
+checksum=878a4194ecf48c48a89ba0a07fcce6d1ad27b3b1c9ec86e52160960ed0199f89
+
+do_build() {
+	cd ${wrksrc}/trunk/src/
+	make clean
```

Comment:
Why do we need to `make clean` on fresh copy?

Reply:
well their tarball isn't clean